build/Dockerfile.nginx

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.4
+FROM nginx:1.25.1-alpine
```

Comment:
Can we also update the main README because now we bind to a specific version of NGINX?